apps/lib/apps.c

```diff
@@ -717,33 +719,36 @@ int load_key_certs_crls(const char *uri, int maybe_stdin,
         *ppubkey = NULL;
     if (pcert != NULL)
         *pcert = NULL;
+    if (failed == NULL) {
+        BIO_printf(bio_err, "Internal error: nothing to load from %s\n",
+                   uri != NULL ? uri : "<stdin>");
```

Comment:
How do you know there's nothing to load before you tried to load?

Reply:
Because the called did not provide any result pointer, so the loading request is effectively void.
That's why I prepended this with "Internal error:".

Reply:
The message is still misleading.  "nothing to load into from %s" might be better

Reply:
Indeed - done.